Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```diff
@@ -397,6 +409,9 @@
     items:
     - Pen
     - BooksBag
+    - HandLabeler
+    - Lighter
+    - DrinkShaker
```

Comment:
what

Reply:
Hand labeler makes sense, lighter is for... arson? Burning papers. :trollface: 

Shaker should probably be in it's own bartending module
but that will probably just take the bartenders job, but also it seems fun for the service borg to be able hold reagents and transfer reagents so yeah I guess

Reply:
But how do you pour something into that shaker?

Reply:
add a syringe to the *literacy* module :trollface:

Reply:
i forgor to rename it to service module